.github/workflows/superlinter.yml

```diff
@@ -0,0 +1,21 @@
+name: Super-Linter CI
+on: [push, pull_request]
+jobs:
+  build-lint-test:
+    name: Build, lint and test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: yarn install
+    - run: yarn build
```

Comment:
Do we need to use build? It didn't generate files in repo and linters / test uses src folder but probably takes a lot of time.
Or am I missing something?